app/src/main/res/layout/activity_settings.xml

```diff
@@ -393,6 +393,48 @@
                     android:text="@string/show_incoming_calls_full_screen" />
 
             </RelativeLayout>
+
+            <include
+                android:id="@+id/settings_migration_divider"
+                layout="@layout/divider" />
+
+            <TextView
+                android:id="@+id/settings_migration_section_label"
```

Comment:
the label doesnt change its color properly, add it in onResumes list of section titles